front/app/modules/commercial/matomo/getProjectId.ts

```diff
@@ -11,16 +11,20 @@ import { Subscription } from 'rxjs';
 
 export const getProjectId = async (path: string) => {
   if (isProjectPage(path)) {
-    const slug = extractProjectSlug(path);
-    if (!slug) return null;
+    if (isOnAdminProjectPage(path)) {
+      return extractProjectIdOrSlug(path);
```

Comment:
Maybe add a comment here to explain we have an id in the admin and a slug for citizens?

Reply:
Here I was more writing code for the actual behaviour right now. I think we use slugs in the front office because they are more user friendly but I can't 100% say that is why the team made that decision. Do you know why? It wouldn't be nice a wrong comment

Reply:
> in the front office because they are more user friendly

At least slugs are not used in the admin AFAIK. We could add it without explaining the why.